contracts/src/FROSTCoordinator.sol

```diff
@@ -24,13 +26,23 @@ contract FROSTCoordinator {
         FROSTNonceCommitmentSet.T nonces;
         GroupParameters parameters;
         Secp256k1.Point key;
+        FROSTFinalization.T finalizations;
+    }
+
+    enum GroupStatus {
+        UNINITIALIZED,
+        COMMITTING,
+        SHARING,
+        CONFIRMING,
+        FINALIZED
```

Comment:
I think we need an additional `COMPROMISED` state for when there are `threshold` complaints to the same participant.